ansible/roles/virtualenv/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: virtualenv | check variables
+  fail:
+    msg: "Invalid value of virtualenv_user: {{ virtualenv_user }}"
+  when: not virtualenv_user or not virtualenv_user|length>0
+  fail:
+    msg: "Invalid value of virtualenv_path: {{ virtualenv_path }}"
+  when: not virtualenv_path or not virtualenv_path|length>0
+
+- name: virtualenv | install python tools
+  become: yes
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - python-pip
+    - python-virtualenv
+
+- name: virtualenv | pip compilation requirements {{ virtualenv_pip_rpm_dependencies }}
+  become: yes
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ virtualenv_pip_rpm_dependencies }}"
+
+- name: virtualenv | pypi {{ virtualenv_pip_dependencies }}
+  become: yes
+  become_user: "{{ virtualenv_user }}"
```

Comment:
Can you make `virtualenv_user` default to `root`, since that's what you'd probably use in production.